app/javascript/widget/components/ChatFooter.vue

```diff
@@ -1,15 +1,25 @@
 <template>
   <footer
     v-if="!hideReplyBox"
-    class="shadow-sm bg-white mb-1 z-50 relative"
-    :class="{ 'rounded-lg': !isWidgetStyleFlat }"
+    class="relative z-50 mb-1"
+    :class="{
+      'rounded-lg': !isWidgetStyleFlat,
+      'pt-2.5 shadow-[0px_-20px_20px_1px_rgba(0,_0,_0,_0.05)] dark:shadow-[0px_-20px_20px_1px_rgba(0,_0,_0,_0.15)] rounded-t-none':
+        hasReplyTo,
+    }"
   >
+    <footer-reply-to
+      v-if="hasReplyTo"
+      :in-reply-to="inReplyTo"
+      @dismiss="inReplyTo = null"
+    />
     <chat-input-wrap
+      class="bg-white shadow-sm"
```

Comment:
@scmmishra Do we need this `bg-white` here? I think the component already has.

Reply:
Yeah, we can remove it

```suggestion
      class="shadow-sm"
```